twirp/client.py

```diff
@@ -22,8 +22,14 @@ def _make_request(self, *args, url, ctx, request, response_obj, **kwargs):
                 response = response_obj()
                 response.ParseFromString(resp.content)
                 return response
-            else:
+            try:
                 raise exceptions.TwirpServerException.from_json(resp.json())
+            except:
```

Comment:
This will always catch the exception. Including `exceptions.TwirpServerException`

As a result, we will always only get either `Unavailable` or `Unknown`

Reply:
Oops, I was catching the specific error in my local patch, not sure how I missed adding it to the PR. Fixed it, thanks!

Reply:
Thanks it seems to work now!